codec-http2/src/test/java/io/netty/handler/codec/http2/Http2MultiplexCodecTest.java

```diff
@@ -170,6 +170,16 @@ public void inboundDataFrameShouldEmitWindowUpdateFrame() {
         verifyFramesMultiplexedToCorrectChannel(inboundStream, inboundHandler, 2);
     }
 
+    @Test
+    public void unhandledHttp2FramesShouldBePropagated() {
+        ByteBuf content = UnpooledByteBufAllocator.DEFAULT.buffer(8).writeLong(0);
+        Http2PingFrame decodedFrame = new DefaultHttp2PingFrame(content);
+
+        codec.onHttp2Frame(decodedFrame);
+        Http2PingFrame receivedPing = parentChannel.readInbound();
+        assertSame(receivedPing, decodedFrame);
+    }
```

Comment:
IIUC we need a `decodedFrame.release();` after the assert.

Reply:
Should be fixed now.